rest-service/manager_rest/config.py

```diff
@@ -184,11 +184,11 @@ def load_from_db(self):
             setattr(self, conf_value.name, conf_value.value)
 
         stored_brokers = session.query(models.RabbitMQBroker).all()
-        broker_hosts = [broker.host for broker in stored_brokers]
+        self.amqp_host = [b.host for b in stored_brokers]
+        self.amqp_management_host = [b.management_host for b in stored_brokers]
         for broker in stored_brokers:
-            # currently, there's going to be only one rabbitmq
-            self.amqp_host = broker_hosts
```

Comment:
oh look, this was already a list

the list is already handled by the amqp client, and it is even copied:
https://github.com/cloudify-cosmo/cloudify-common/blob/208d78287eacff09a3e160368136e99ae8eaeaf9/cloudify/amqp_client.py#L147-L150